.evergreen/run-custom-csfle-tests.sh

```diff
@@ -10,6 +10,11 @@ if [ -z ${CSFLE_KMS_PROVIDERS+omitted} ]; then echo "CSFLE_KMS_PROVIDERS is unse
 export AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID}
 export AWS_SECRET_ACCESS_KEY=${AWS_SECRET_ACCESS_KEY}
 export CSFLE_KMS_PROVIDERS=${CSFLE_KMS_PROVIDERS}
+echo "csfle CRYPT_SHARED_LIB_PATH: $CRYPT_SHARED_LIB_PATH"
```

Comment:
It's now gone: https://evergreen.mongodb.com/task_log_raw/mongo_node_driver_next_ubuntu1804_custom_dependency_tests_run_custom_csfle_shared_lib_tests_pinned_commit_patch_c70c69edb7b51aeac143aa2eb73aaaf465c41572_62f12690d1fe07159c0476cb_22_08_08_15_06_57/0?type=T#L270